bundles/org.openhab.binding.kodi/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -5,7 +5,7 @@
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
 	<!-- Kodi Thing Type -->
-	<thing-type id="kodi" extensible="shownotification">
+	<thing-type id="kodi" extensible="pvr-open-tv,pvr-open-radio,shownotification">
```

Comment:
Are these channels made to be there multiple times? Can the handler handle it correctly?
If so, you should also adapt the README accordingly.

Reply:
Btw, this change does not really fit to the title of this PR...

Reply:
I commited them by mistake. I removed them now.